assets/src/edit-story/style.css

```diff
@@ -48,17 +52,8 @@ body.edit-story #wpfooter {
   display: none;
 }
 
-body.edit-story #edit-story .components-spinner {
```

Comment:
Why was this removed?

Reply:
We don't use `components-spinner` anywhere